features/other_a/invite_queue.feature

```diff
@@ -106,8 +106,7 @@ Feature: Invite queue management
       And I follow "Invitations"
       And I fill in "track_invitation_invitee_email" with "test@archiveofourown.org"
       And I press "Search" within "form.invitation.simple.search"
-    Then I should see "Sender queue"
-    When I follow "copy and use"
+    When I follow "Copy and use"
```

Comment:
For steps with the same type (i.e. `When`), we use `And` and indent them by two spaces:
```suggestion
      And I follow "Copy and use"
```
You can see an example of this in the list of steps above